test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -133,3 +157,38 @@ reuse_topic requires that sink input dependencies are sources, materialize.publi
   INTO KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'output13-view-${testdrive.seed}'
   WITH (reuse_topic=true, consistency_topic='output13-view-consistency-${testdrive.seed}')
   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
+
+# ensure that the sink works with log compaction enabled on the consistency topic
+
+$ kafka-create-topic topic=compaction-test-input-consistency
+$ kafka-create-topic topic=compaction-test-input
+#
+# already create topic with compaction
```

Comment:
im not sure i know what this topic means specifically, the already is throwing me off

Reply:
ah, this means "create the topic now, with the known compaction setting, instead of letting materialize create it as part of the CREATE SINK". Because then we can't change the compaction later. Unless we want to add a sink option that sets this, sth like `consistency_compaction=true`.